### DISCUSSION TITLE
Support easier navigation across global search or references results

### DISCUSSION THREAD

**Discussion by jlebon** [helix-editor/helix#5513] [Ideas]
For commands which output a list of results, such as the global search results window (`<space>/`) and the references window (`gr`), it'd be great to have `[`/`]` shortcuts to go to the previous/next result in the list without having to go back to the picker with `<space>'` each time.

**Comment by jlebon** [comment]
Ahhh sorry, I missed using the enhancement template for this which points to the Discussions section. Let me know if I should open this there instead. (Or maybe there's a way to transfer an open issue.)

**Comment by the-mikedavis** [comment]
See https://github.com/helix-editor/helix/pull/3360 which aims to add this

I believe it's possible to do this as a macro keybind though (#1383) without adding new commands

**Reply by jlebon** [reply]
Thanks! Good to see it's already on the radar. Will keep an eye on those PRs.
